NYPLAudiobookToolkit/UI/ScrubberView.swift

```diff
@@ -251,14 +264,27 @@ final class ScrubberView: UIView {
 
     override func updateConstraints() {
         self.barWidthConstraint?.constant = self.state.progressLocationFor(self.progressBarWidth)
-        self.progressBar.backgroundColor = self.state.progressColor
-        self.gripper.backgroundColor = self.trimColor
         self.labelWidthConstraints.forEach { (constraint) in
             constraint.constant = self.state.progress.labelWidth
         }
         super.updateConstraints()
     }
     
+    private func updateColors() {
+        if #available(iOS 12.0, *),
+           UIScreen.main.traitCollection.userInterfaceStyle == .dark {
+            self.progressBackground.backgroundColor = NYPLColor.progressBarBackgroundColor
+            self.state.progressColor = NYPLColor.actionColor
+        } else {
+            self.progressBackground.backgroundColor = UIColor.darkGray
+            self.state.progressColor = .black
+        }
+        
+        self.progressBar.backgroundColor = self.state.progressColor
+        self.gripper.backgroundColor = self.trimColor
+        self.layoutIfNeeded()
```

Comment:
same question as above here. Basically doing layout just for the colors seems excessive, if this expensive operation can be avoided it would be great.